scene/resources/default_theme/SCsub

```diff
@@ -2,4 +2,20 @@
 
 Import("env")
 
+import os
+import os.path
+from platform_methods import run_in_subprocess
+import default_theme_builders
+
 env.add_source_files(env.scene_sources, "*.cpp")
+
+path = env.Dir(".").abspath
+
+env.Depends(
+    "#scene/resources/default_theme/default_font.gen.h", path + "/../../../thirdparty/fonts/OpenSans_SemiBold.ttf"
```

Comment:
Can't you use `#thirdparty/fonts/OpenSans_SemiBold.ttf` instead of a relative path?